pkg/deploy/graph/analysis/dc.go

```diff
@@ -50,21 +55,40 @@ dc:
 					continue dc
 				}
 
-				// 2. Build config points to image stream tag of interest
 				if bcNode, points := buildPointsToTag(g, uncastIstNode); points {
-					markers = append(markers, osgraph.Marker{
-						Node:         uncastDcNode,
-						RelatedNodes: []graph.Node{uncastIstNode, bcNode},
+					// A build config points to the non existent tag but the latest build has failed.
+					// Since the tag isn't yet populated, there hasn't been any successful build so far.
+					if buildNode, failed := latestBuildFailed(g, bcNode); failed {
```

Comment:
I mean, wouldn't it make more sense to move this entire check to `pkg/build/graph/analysis` and make the marker there?  Both the error and the help make sense there.  The other marker (TagNotAvailable), will still make sense too, but that way I'll get help when all I have are builds or chained builds.